openpype/plugins/publish/extract_colorspace_data.py

```diff
@@ -0,0 +1,47 @@
+import pyblish.api
+from openpype.pipeline import publish
+
+
+class ExtractColorspaceData(publish.ExtractorColormanaged):
```

Comment:
### Warning

This PR has broken publishing in Maya for a project that has NO color space config set due to this `ExtractColorspaceData` node.

I cannot publish e.g. a `pointcache` instance:

```python
Traceback (most recent call last):
  File "S:\openpype\OpenPype\.venv\lib\site-packages\pyblish\plugin.py", line 522, in __explicit_process
    runner(*args)
  File "S:\openpype\OpenPype\openpype\plugins\publish\extract_colorspace_data.py", line 46, in process
  File "S:\openpype\OpenPype\openpype\pipeline\publish\publish_plugins.py", line 376, in set_representation_colorspace
    colorspace_settings = self.get_colorspace_settings(context)
  File "S:\openpype\OpenPype\openpype\pipeline\publish\publish_plugins.py", line 331, in get_colorspace_settings
    anatomy_data=anatomy_data
  File "S:\openpype\OpenPype\openpype\pipeline\colorspace.py", line 365, in get_imageio_config
    "No OCIO config found in settings. It is "
FileExistsError: No OCIO config found in settings. It is either missing or there is typo in path inputs
```

This also goes to show this Extractor is INVALID.
It shouldn't enforce injecting colorspaceData for representations which are NOT a image/video/pixel format like e.g. a pointcache.